deploy/crds/contrail.juniper.net_memcacheds_crd.yaml

```diff
@@ -4,15 +4,21 @@ metadata:
   name: memcacheds.contrail.juniper.net
 spec:
   additionalPrinterColumns:
+  - JSONPath: .status.replicas
+    name: Replicas
+    type: integer
+  - JSONPath: .status.readyReplicas
+    name: Ready_Replicas
+    type: integer
+  - JSONPath: .status.endpoint
+    name: Endpoint
+    type: string
```

Comment:
Nit: maybe, instead of two columns, there could be one with readyReplicas/replicas? Same style as is printed for core k8s resources, like Deployments, pods, etc.?
I think it can be done purely in the JSONPath (not 100% sure), like:

>  \- JSONPath: ".status.readyReplicas/.status.replicas"
>     name: Ready
>     type: string

Reply:
I will try that

Reply:
I think only "simple JSON Paths" are supported